jobs/syncjob.h

```diff
@@ -99,20 +99,14 @@ namespace QMatrixClient
     class SyncJob: public BaseJob
     {
         public:
-            SyncJob(ConnectionData* connection, QString since=QString());
+            SyncJob(ConnectionData* connection, QString filter, int timeout,
```

Comment:
I'd prefer to have "since" before filter and timeout, as I think it is a more commonly used parameter. Also, I'd give filter and timeout a default value ({} and -1).